src/java.base/aix/native/libnio/MappedMemoryUtils.c

```diff
@@ -218,3 +218,19 @@ Java_java_nio_MappedMemoryUtils_force0(JNIEnv *env, jobject obj, jobject fdo,
         JNU_ThrowIOExceptionWithMessageAndLastError(env, "msync with parameter MS_SYNC failed");
     }
 }
+
+#define FD "Ljava/io/FileDescriptor;"
+
+static JNINativeMethod methods[] = {
+    {"isLoaded0", "(JJJ)Z",             (void *)&MMU_isLoaded0},
+    {"load0",     "(JJ)V",              (void *)&MMU_load0},
+    {"unload0",   "(JJ)V",              (void *)&MMU_unload0},
+    {"force0",    "(" FD "JJ)V",        (void *)&MMU_force0},
+};
```

Comment:
I did a pass over this and I think it looks okay.

Do the MMU_* functions still need to be jni-exported? In any case, probably better to rename them to MemoryMemoryUtils_* to avoid anything looking at symbols and thinking they are something to do with the memory management unit :-)

It looks most of the signatures for this functions are misaligned with edits.

Reply:
> Do the MMU_* functions still need to be jni-exported?

I've now dropped JNIEXPORT, but kept JNICALL, as that is used to set `__stdcall` (at least on Windows).